src/test/java/com/mate/fruitshop/strategy/BalanceHandlerTest.java

```diff
@@ -0,0 +1,47 @@
+package com.mate.fruitshop.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import com.mate.fruitshop.db.Storage;
+import com.mate.fruitshop.model.FruitEntry;
+import com.mate.fruitshop.model.Transaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static final FruitEntry STORAGE_ENTRY =
+            new FruitEntry("banana", 5);
+    private static final int FIRST_INDEX = 0;
+    private static BalanceHandler balanceHandler;
+    private static Transaction balanceTransaction;
+
+    @Before
+    public void setUp() {
+        balanceHandler = new BalanceHandler();
+        balanceTransaction = new Transaction(Transaction.Operation.RETURN,
+                "banana", 10);
+    }
+
+    @After
+    public void tearDown() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void process_newFruit_Ok() {

```

Comment:
```suggestion
    public void process_NewFruit_Ok() {
```
check all tests in all classes and use the same naming pattern